step.yml

```diff
@@ -1,6 +1,57 @@
-title: Android Build
-summary: Builds your Android project with gradle.
-description: Builds your Android project with gradle.
+itle: Android Build
+summary: Builds your Android project with Gradle.
+description: |-
+  
+  The Step builds your Android project on Bitrise with Gradle commands: it installs all dependences that are listed in the project's `build.gradle` file, and builds and exports either an APK or an AAB. 
+  Once the file is exported, it is available for other Steps in your Workflow.
+  
+  You can select the module and the variant you want to use for the build.
+  
+  ### Configuring the Step 
+  
+  1. Make sure the **Project Location** input points to the root directory of your Android project.
+  1. In the **Module** input, set the module that you want to build.
+     
+     You can find the available modules in Android Studio. If you leave the input empty, the Step will use the default module that is set when you add your project to Bitrise. 
+
+  1. In the **Variant** input, set the variant that you want to build. 
+     
+     You can find the available variants in Android Studio. If you leave the input empty, the Step will use the default variant that is set when you add your project to Bitrise. 
```

Comment:
That's not true.
The default value of the Variant and the Module is empty: `""`
https://github.com/bitrise-steplib/bitrise-step-android-build/pull/21/files#diff-6bc88361fc7fca6b922fde7f3243d0e3815d0584b5493c16dc45633b36172529R84

https://github.com/bitrise-steplib/bitrise-step-android-build/pull/21/files#diff-6bc88361fc7fca6b922fde7f3243d0e3815d0584b5493c16dc45633b36172529L41